saic-java-mqtt-gateway/src/main/java/net/heberling/ismart/mqtt/VehicleHandler.java

```diff
@@ -488,10 +488,10 @@ public void handleMQTTCommand(String topic, MqttMessage message) throws MqttExce
               sendACCommand((byte) 0, (byte) 0);
               break;
             case "on":
-              sendACCommand((byte) 2, (byte) 8);
+              sendACCommand((byte) 2, (byte) (vehicleState.getRemoteTemperature() - 14));
```

Comment:
This should be an own method, that is returning the correct value depending on the car model. I believe all current European cars are using this same offset, but older cars in other regions use different values.

Reply:
Of course, just want to test it first and see if it works. Problem is I only found this mapping byte values starting at 2 for 16°, but you are right, we will need to at least set the option to configure this...